deploy/kubernetes/minikube/zookeeper.yaml

```diff
@@ -30,13 +30,31 @@ metadata:
 spec:
   serviceName: zookeeper
   replicas: 1
+  selector:
+    matchLabels:
+      app: heron
+      component: zookeeper
   template:
     metadata:
       labels:
         app: heron
         component: zookeeper
 
     spec:
+      # Make sure multiple pods of ZK don't get scheduled on the
+      # same node, unless there are no other available nodes
+      affinity:
+        podAntiAffinity:
```

Comment:
Do we have podAntiAffinity set for the rest of the running components?  If not, do you think it would make sense to specify anitAffinity on the other tools in the system?